README.md

```diff
@@ -81,6 +79,25 @@ cp .env.example .env
 
 After filling in the values in `.env`, you are ready to run the project!
 
+### Authentication Setup with Clerk
+
+To setup authentication with Clerk, follow the guide [here](https://clerk.com/docs/quickstarts/setup-clerk).
+
+TLDR: You need to make an account and update the following environment variables in `.env`. Note: you do not need to modify NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY.
```

Comment:
update this to say "join the Centre for Dreams organization" or something similar